docker-compose.yml

```diff
@@ -33,7 +33,6 @@ services:
       - ./packages/venia-concept/static:/usr/src/app/packages/venia-concept/static:rw
       - ./packages/venia-ui/.storybook:/usr/src/app/packages/venia-ui/.storybook:rw
       - ./packages/venia-ui/lib:/usr/src/app/packages/venia-ui/lib:rw
-      - ./packages/venia-ui/templates:/usr/src/app/packages/venia-ui/templates:rw
```

Comment:
For starters, I might have scope creeped this a bit but as far as I can tell we no longer use the `templates`, opting instead to use `venia-concept/template.html`. @zetlen @revanth0212 can ya'll confirm this?

Reply:
You are right. We got rid of mustache templates in favor of Webpack's HTML Plugin which utilized template.html to render the main HTML file.